### PR TITLE
added-javascript

### PR DESCRIPTION
**Pull request by jesberman**
I added a JavaScript file that creates the requested functionality.

### CODE REVIEW

**Comment by octovinh** [comment]
Thanks for the PR, Jesse!

What are the benefits of adding jQuery to the page? What are the benefits of leaving it off the page?

File: assets/javascript/logic.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+//Begins a function meant to display the user's input within the "#page" div as well as
+//indicating the number of seconds between the user's click and the moment the page
+//was first loaded.  The first step is to identify the "Post" button by its Tag name, 
+//and then add an event listener to it that listens for a "click."
+document.getElementsByTagName("INPUT")[1].addEventListener("click", function () {
+//Instructs the function to not perform whatever action it normally would have performed.
+    event.preventDefault();
+    //declares the "userInput" variable, and defines its value as whatever the user
+    //has written within the first "input" field.
+    let userInput = document.getElementsByTagName("INPUT")[0].value;
```
Comment (LorenzoGangi) [inline assets/javascript/logic.js]: Why do the query again? 
```
let inputs = document.getElementsByTagName("INPUT")
inputs[1].addEventListener( .....
    let userInput = inputs[0].value;
```

One less DOM query.

**Comment by jesberman** [comment]
Your very welcome, Vinh.  To answer your questions, my understanding is
that some of the benefits of using jQurey include the fact that it is
lightweight and allows for faster loading, it allows for cross browser
compatibility, and it promotes simplicity with its easy to understand
syntax.

Some disadvantages to using jQuery include how their are apparently a large
number of open source jQuery plugins that are of low coding quality, and
the fact that jQuery can often be written in an inefficient manner.

Looking forward to speaking with you soon.

-Jesse Berman

**Comment by jesberman** [comment]
Hi Vinh, I wanted to thank you for taking the time to interview me today.
As a front-end developer I know there is much I can do to contribute to
Ontraport, and I very much hope I can have the chance to prove myself.  If
you have any additional questions for me, please feel free to ask.  Looking
forward to hearing from you.

Sincerely,

-Jesse Berman
https://jesberman.github.io/

**Pull request closed without merging**